lti_consumer/static/js/xblock_studio_view.js

```diff
@@ -72,33 +72,36 @@ function LtiConsumerXBlockInitStudio(runtime, element) {
      *
      *  new - Show all the LTI 1.1/1.3 config fields
      *  database - Do not show the LTI 1.1/1.3 config fields
-     *  external - Show only the External Config ID field
+     *  external - Show all LTI 1.3 config fields except LTI 1.3 tool fields
```

Comment:
[We are not adding field overrides in this PR](https://github.com/openedx/xblock-lti-consumer/pull/390#discussion_r1316979821), so I believe we should remove the changes from this file.

Reply:
I modified this file to remove all LTI 1.3 fields except external ID and modified this comment on the latest commit.